core-web/libs/dotcms-models/src/lib/dot-experiments-constants.ts

```diff
@@ -221,7 +221,14 @@ export const AllowedActionsByExperimentStatus: Record<
         DotExperimentStatus.ENDED,
         DotExperimentStatus.ARCHIVED
     ],
-    ['archive']: [DotExperimentStatus.ENDED]
+    ['archive']: [DotExperimentStatus.ENDED],
+    ['addToBundle']: [
```

Comment:
I wonder if this should be not retrieve from the backend, any new state or action will need to be replicated here